File: laboratorioMombelli/esempio.js
let game = {
    nome : 'Minecraft Java Edition',
    platform : 'PC',
    publisher: 'Mojang Studios',
    year : 2011
};

let gameJSON = JSON.stringify(game); // converte l'oggetto in una stringa JSON
console.log(game);
console.log(gameJSON); 

let game2 = JSON.parse(gameJSON); // converte la stringa JSON in un oggetto JavaScript
console.log(game2);

File: laboratorioMombelli/esempio.py
import json

game = {
    "nome" : 'Minecraft Java Edition',
    "platform" : 'PC',
    "publisher": 'Mojang Studios',
    "year" : 2011
}

game_json = json.dumps(game)

print(game_json)

with open('game.json',encoding= 'utf-8', mode='w') as f:
    json.dump(game_json, f, indent=4)

with open('game.json', encoding='utf-8', mode='r') as f:
    object = json.load(f)

print(object)